server/util/twilio.js

```diff
@@ -68,3 +61,10 @@ export function mutateCallConnectContact(callSid, idCollection) {
       url: `${DEV_PATH}/users/${user_id}/campaigns/${campaign_id}/calls/${call_id}/connect`
     });
 }
+
+export function hangUpContactCall(callSid, userId, campaignId) {
+  return twilioClient.calls(callSid)
+    .update({
+      url: `${DEV_PATH}/users/${userId}/campaigns/${campaignId}/calls/bridge`
```

Comment:
Hope we've confirmed this is where the bridge Twiml lives? Should be identical to the redirect at the end of the call twiml.